db/migrate/20160415121724_create_cloud_services.rb

```diff
@@ -0,0 +1,21 @@
+class CreateCloudServices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :cloud_services do |t|
+      t.string :ems_ref
+      t.string :source
+      t.string :binary
+      t.string :hostname
+      t.string :status
+      t.boolean :scheduling_disabled
+      t.string :scheduling_disabled_reason
+      t.string :zonename
```

Comment:
Is this `AvailabilityZone#name`?  If so, remove it from here.  This can be accessed via `cloud_service.availability_zone.name`.

Same reason [as above](https://github.com/ManageIQ/manageiq/pull/8093/files#r60257227).